src/backend/distributed/shared_library_init.c

```diff
@@ -670,6 +670,21 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.allow_unsafe_locks_from_workers",
+		gettext_noop("Enables acquiring a distributed lock from a worker "
+					 "when the coordinator is not in the metadata"),
+		gettext_noop("Set to false by default. If set to true, enables "
```

Comment:
maybe briefly touch why? Because it might cause concurrent operations from the coordinator and the workers do not follow the lock ordering.